lib/lotus/helpers/html_helper.rb

```diff
@@ -184,8 +193,13 @@ module HtmlHelper
       #
       # @see Lotus::Helpers::HtmlHelper
       # @see Lotus::Helpers::HtmlHelper::HtmlBuilder
-      def html
-        HtmlBuilder.new
+      # @see Lotus::Helpers::HtmlHelper::HtmlFragment
+      def html(&blk)
+        if block_given?
+          HtmlBuilder.new.fragment(&blk)
```

Comment:
@huynhquancam Why not just simplify this call with the following syntax?

```ruby
def html(&blk)
  HtmlBuilder.new(&blk)
end
```

And then in `HtmlBuilder`:

```ruby
def initialize(&blk)
  # ...
  instance_eval(&blk) if block_given?
end
```